packages/pigeon/pigeons/core_tests.dart

```diff
@@ -56,6 +56,18 @@ abstract class HostIntegrationCoreApi {
   @ObjCSelector('createNestedObjectWithString:')
   AllTypesWrapper createNestedString(String string);
 
+  /// Returns passed in arguments of multiple types.
+  @ObjCSelector('sendMultipleTypesABool:anInt:aString:')
+  AllTypes sendMultipleTypes(bool aBool, int anInt, String aString);
+
+  /// Returns passed in int.
+  @ObjCSelector('echoInt:')
+  int? echoInt(int anInt);
```

Comment:
Are these nullable just to get a mix of nullable and non -nullable tests? Since we eventually probably want to fully cover all types both nullable and not (for C++, and ObjC if we change use generation), I would suggest were make the arguments match, and for nullable tests put nullable in the title for nullable tests. I.e., have `int echoInt(int)` and `int? echoNullableInt(int?)`.

I forgot nullable types on the list in the issue; I'll add that now.